Sources/Library/Library.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+/// An example of a struct with only `Sendable` properties.
+///
+/// This type would be implicitly-`Sendable` within its defining module, but
+/// because it is public the `Sendable` conformance must be explicit.
+public struct ColorComponents {
```

Comment:
The comment here is a bit confusing, because we're saying "it would be ... but ... so ... it must be explicitly Sendable" -- but then it's not actually Sendable. So we're explaining it "must"  but then we don't follow that must.

It makes me think "is this missing a Sendable, or not making it sendable on purpose..."

Maybe reword that: 

> this type is NOT Sendable because it is public. If we want a public type to be Sendable, we must annotate it explicitly.

?